### ISSUE TITLE
The Angular Material Schematic workflow failed with Jest

### ISSUE THREAD

**Issue by ivibe** [nrwl/nx#868]
OS: Windows 10 64bit
@nrwl/schematics@7.0.0
VS Code 1.28.2
Typescript 3.1.3

When I try to add Angular Material to the project with Jest as Unit Test Runner.
```ng add @angular/material --project=myapp```
I get the next error
```Your project is not using the default builders for "test". The Angular Material schematics can only be used if the original builders from the Angular CLI are configured.```

Apparently Angular CLI doesn't like
```
...
"test": {
          "builder": "@nrwl/builders:jest",
...
```
instead of
```
...
"test": {
          "builder": "@angular-devkit/build-angular:karma",
...
```

What your thoughts about that?

**Comment by chauey** [comment]
yeah i ran into the same problem and stuck, looking around for solutions

**Comment by ivibe** [comment]
In addition there's also an issue when trying to use Angular Material component schematics (address-form, navigation, dashboard etc.).

For example, I get next error message
```
Selector (apps/main-form) is invalid.
```
when trying to execute the next command
```
ng g @angular/material:address-form form --project=main
```
Apparently Angular Material doesn't like slash in the name of the prefix, set in angular.json

The workaround is using a --prefix option in the command
```
ng g @angular/material:address-form form --project=main --prefix=main
```

**Comment by FrozenPandaz** [comment]
I have asked for the error to be downgraded to a warning:

https://github.com/angular/material2/issues/14176

**Comment by FrozenPandaz** [comment]
Update:

This issue will be fixed by https://github.com/angular/material2/pull/14203.

Thank you @DevVersion
